storybookjs/storybook
# Web-components: Support lit 2 with back-compat

Issue: refs #13489

## What I did
Use split versions of `lit-html` and `lit-element` pointing to the pre-`lit@2.0` versions that support the `isTemplateResult()` helper.

I checked out the `@storybook/lit` build, and I'm not sure what it's over all goals are, but having it separate seems like a weird play in the context of the x-compatibility story of web components, but happy to revisit if not. 

## How to test

- Is this testable with Jest or Chromatic screenshots? By testing the `@storybook/web-components` examples.
- Does this need a new example in the kitchen sink apps? Uses the `@storybook/web-components` examples.
- Does this need an update to the documentation? Possibly?

If your answer is yes to any of these, please make sure to include it in your PR.
